examples/java/pom.xml

```diff
@@ -432,6 +424,11 @@
 
     <dependency>
       <groupId>org.apache.beam</groupId>
+      <artifactId>beam-runners-direct-java</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>org.apache.beam</groupId>
       <artifactId>beam-sdks-java-io-google-cloud-platform</artifactId>
```

Comment:
(Thinking ahead to the archetype port of this... what is the plan for linking in the different runners?)